platform/uwp/export/export_plugin.h

```diff
@@ -85,6 +85,11 @@ static const char *uwp_device_capabilities[] = {
 	nullptr
 };
 
+// Optional environment variables for defining confidential information. If any
+// of these is set, they will override the values set in the credentials file.
+const String ENV_UWP_SIGNING_CERT = "GODOT_UWP_SINGING_CERTIFICATE";
+const String ENV_UWP_SIGNING_PASS = "GODOT_UWP_SINGING_PASSWORD";
```

Comment:
```suggestion
const String ENV_UWP_SIGNING_CERT = "GODOT_UWP_SIGNING_CERTIFICATE";
const String ENV_UWP_SIGNING_PASS = "GODOT_UWP_SIGNING_PASSWORD";
```

Reply:
🤦‍♂️

Reply:
It's such a common typo that I decided to go through all defines just to spot if it was made, and indeed :microphone: